wp-includes/rest-api/auth/jwt/JWT.php

```diff
@@ -0,0 +1,379 @@
+<?php
+
+namespace Firebase\JWT;
```

Comment:
Anybody know the policy on external dependencies like this, in terms of where they should go in the codebase? Most of them go into `/wp-includes/[ project ]`, looking at SimplePie, Requests, etc. I'd expect this should be followed here

Reply:
I've moved it 👍